code/cross-chain-tx/scripts/l1tol2tx.ts

```diff
@@ -0,0 +1,90 @@
+import { ethers, network } from 'hardhat';
+import GREETER_ABI_JSON from '../artifacts-zk/contracts/Greeter.sol/Greeter.json';
+
+// ANCHOR: contract-address
+// L2 contract address on ZKsync sepolia
+const L2_CONTRACT_ADDRESS = '0x543A5fBE705d040EFD63D9095054558FB4498F88';
```

Comment:
remove address?

Reply:
This is needed for users to not need to deploy a contract on sepolia first when running on sepolia